ADVANCED.md

```diff
@@ -653,6 +654,64 @@ We have more discussion and examples [in our issue here](https://github.com/sw-y
 
 </details>
 
+# Section 4: @types/react and @types/react-dom APIs
+
+The `@types` typings export both "public" types meant for your use as well as "private" types that are for internal use.
+
+## `@types/react`
+
+[Link to `.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts)
+
+**Namespace: React**
+
+Most Commonly Used Interfaces and Types
+
+- `ReactNode` - anything that is renderable *inside* of JSX, this is NOT the same as what can be returned from a component!
+- `Component` - base class of all class-based components
+- `PureComponent` - base class for all class-based optimized components
+- `FC`, `FunctionComponent` - a complete interface for function components, often used to type external components instead of typing your own
+- `CSSProperties` - used to type the `css` prop
```

Comment:
```suggestion
- `CSSProperties` - used to type style objects
```
This may be a little bit more general since `css` prop is only used in some css-in-js solutions and inline styles are in general discouraged.